src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Microsoft.VisualStudio.LanguageServerClient.Razor.csproj

```diff
@@ -16,8 +16,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="StreamJsonRpc" Version="$(StreamJsonRpcPackageVersion)" />
```

Comment:
Without these we run into Version conflicts. Turns out @TanayParikh had to explicitly add it on this particular project anyway in https://github.com/dotnet/aspnetcore-tooling/pull/3116/files.